docs/metrics.md

```diff
@@ -66,3 +66,9 @@ Triggered whenever an attempt is acted upon. Includes:
 - `cm5`
 - `cd1` (for this attempt)
 - `cd5`
+
+## Additional Collection
+The uploaded voice samples and the JSON response containing the suggestions are
+stored together under a random UUID.  There is no identifying information nor
```

Comment:
We do not collect any identifying information or the selected search suggestions.